packages/block-library/src/pullquote/test/edit.native.js

```diff
@@ -45,10 +46,13 @@ describe( 'Pullquote', () => {
 
 		const citationTextInput =
 			within( citationBlock ).getByPlaceholderText( 'Add citation' );
-		typeInRichText( citationTextInput, 'A person', {
-			finalSelectionStart: 2,
-			finalSelectionEnd: 2,
+		typeInRichText( citationTextInput, 'A person' );
+		fireEvent( citationTextInput, 'onKeyDown', {
+			nativeEvent: {},
+			preventDefault() {},
+			keyCode: ENTER,
 		} );
+		selectRangeInRichText( citationTextInput, 2, 2 );
```

Comment:
Nitpick: We don't need to pass the `end` argument as it's populated with `start` value by default.
```suggestion
		selectRangeInRichText( citationTextInput, 2 );
```

https://github.com/WordPress/gutenberg/blob/5c3d93a949faff5062b158378364326064156d18/test/native/integration-test-helpers/rich-text-select-range.js#L14

Reply:
I keep overlooking this. 😄 Resolved in https://github.com/WordPress/gutenberg/pull/53112.